packages/@aws-cdk/aws-dynamodb/test/dynamodb.test.ts

```diff
@@ -159,6 +159,50 @@ describe('default properties', () => {
     );
   });
 
+  test('contributor insights is not enabled', () => {
+    new Table(stack, CONSTRUCT_NAME, {
+      partitionKey: TABLE_PARTITION_KEY,
+      sortKey: TABLE_SORT_KEY,
+    });
+
+    expect(stack).toHaveResource('AWS::DynamoDB::Table',
+      {
+        AttributeDefinitions: [
+          { AttributeName: 'hashKey', AttributeType: 'S' },
+          { AttributeName: 'sortKey', AttributeType: 'N' },
+        ],
+        KeySchema: [
+          { AttributeName: 'hashKey', KeyType: 'HASH' },
+          { AttributeName: 'sortKey', KeyType: 'RANGE' },
+        ],
+        ProvisionedThroughput: { ReadCapacityUnits: 5, WriteCapacityUnits: 5 },
+      },
+    );
+  });
```

Comment:
We got a lot of these negative tests already. Every single one up to your PR, in fact 🙂. So I would safely remove this test.